<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBranchAttr</class>
 <widget class="QDialog" name="DialogBranchAttr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>子块属性</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0" colspan="8">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="MyGraphicsView" name="graphicsView_Term"/>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="1" column="0" colspan="2">
          <widget class="QWidget" name="widget_8" native="true">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QPushButton" name="BtnCancelTermSign_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>消除标记</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="BtnFromUnitImage_2">
              <property name="text">
               <string>从元件库选择</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="BtnFromDisk_2">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="BtnSaveTerm_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QFrame" name="frameTag_Term_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>110</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="CbShowUnitTag">
     <property name="text">
      <string>显示代号</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>项目代号</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="EdDesignation"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="EdUnitTag">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="EdUnitProTag">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QToolButton" name="BtnOk">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>子块代号</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QToolButton" name="BtnUnitAttr">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>元件属性</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QToolButton" name="BtnCancel">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="EdSpurDT"/>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QTableWidget" name="tableWidgetTermInfo">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>端号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>测试代价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>图片路径</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否标注</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>元件代号</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>序    号</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="4" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="LbFunDefine">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="LbFunSymbolView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="CbSymbolPattern">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="BtnReplaceFunSymbol">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>替换</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="BtnProSet">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">widget/mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
